pkg/operator/controller/dns_status.go

```diff
@@ -52,35 +72,96 @@ func computeDNSStatusConditions(dns *operatorv1.DNS, clusterIP string, haveDNSDa
 		}
 	}
 
-	conditions := []operatorv1.OperatorCondition{
-		computeDNSDegradedCondition(oldDegradedCondition, clusterIP, haveDNSDaemonset, dnsDaemonset),
-		computeDNSProgressingCondition(oldProgressingCondition, dns, clusterIP, haveDNSDaemonset, dnsDaemonset, haveNodeResolverDaemonset, nodeResolverDaemonset),
-		computeDNSAvailableCondition(oldAvailableCondition, clusterIP, haveDNSDaemonset, dnsDaemonset),
-		computeDNSUpgradeableCondition(oldUpgradeableCondition, dns),
-	}
+	conditions := []operatorv1.OperatorCondition{}
```

Comment:
This line is superfluous because the next line redeclares `conditions`.

Reply:
removed